code/modules/clothing/spacesuits/rig_modules/rig_modules.dm

```diff
@@ -97,4 +99,24 @@
 
 		//NEED TO GET MAXIMUM AMOUNT OF MOLES WITHOUT GOING OVER 10*ONE_ATMOSPHERE
 		//pressure = total_moles * R_IDEAL_GAS_EQUATION * temperature / volume
-		//pressure_delta = target_moles * 8.314 * sample_temperature / internals.volume
\ No newline at end of file
+		//pressure_delta = target_moles * 8.314 * sample_temperature / internals.volume
+
+
+/obj/item/rig_module/plasma_proof
+	name = "plasma-proof sealing authority"
+	desc = "Brings the suit it is installed into up to plasma environment standards."
+
+/obj/item/rig_module/plasma_proof/activate(var/mob/user,var/obj/item/clothing/suit/space/rig/R)
+	..()
+	if(R.cell.use(250))
```

Comment:
shouldn't this (and the speedboost) be a low, constant powerdrain in process() like the oxygen refiller has?

Reply:
it's a seal, so logically it would only take energy to actually seal the suit. Though it requiring energy at all is a bit strange, since you should be sealed into the suit to begin with.

Reply:
Presumably the difference between a blasma-proof suit and another, otherwise similarly airtight suit, is that the atmos suit is made of a special plasma-repellent material. I'm imagining this module as providing routine cleaning or periodically lubricating the outside of your suit with plasmaphobic fluids.